### PR TITLE
fix: fixed dropdown styling

### LINKED ISSUE #4063: Bug: Workspace dropdown in Add to workspace modal it too big
The workspace dropdown in the Add to workspace modal it too big. The arrow is in the middle of the dropdown.

CleanShot 2024-08-28 at 18 06 37

### PR DESCRIPTION
**Pull request by ryandotfurrer**
I decreased the size of the modal as well as the dropdown in desktop. However, I increased the size on mobile to make it more inline with the mobile styling.

fix #4063

## Description

This PR fixes bug #4063 by decreasing the size of the dropdown selector on desktop. I also took it upon myself to make the mobile styles more inline by _increasing_ the size of the mobile dropdown. You can see these changes in the video below.

## Related Tickets & Documents
Fixes #4063 

## Mobile & Desktop Screenshots/Recordings
[Watch the video](https://www.youtube.com/watch?v=paWFZ5t_p3E) I made for this fix.
### Desktop
**Before**
Arc Repositories  LetsGetTechnicalgridiron-survivor-000195

**After**
Arc Repositories  LetsGetTechnicalgridiron-survivor-000197

### Mobile
**Before**
Arc Repositories  LetsGetTechnicalgridiron-survivor-000199

**After**
Arc Repositories  LetsGetTechnicalgridiron-survivor-000201

## Steps to QA
Visit any repo of your liking and click the "Add to workspace" button. Please check it in mobile as well.

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [ ] Tier 4

## [optional] What gif best describes this PR or how it makes you feel?
an-older-man-wearing-glasses-and-a-maroon-sweater-is-smiling

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -13,4 +13,4 @@ NEXT_PUBLIC_SENTRY_DSN=https://3f5bb9023ff0407299dd22a6454558f9@o450487248892723
 # For running the API local, use your devices localhost address:
 #
 # NEXT_PUBLIC_API_URL=http://127.0.0.1:3001/v2
-# NEXT_PUBLIC_EXP_API_URL=http://127.0.0.1:3001/v2
+# NEXT_PUBLIC_EXP_API_URL=http://127.0.0.1:3001/v2
```
Comment (nickytonline) [inline .env:16]: refactor (blocking): This change shouldn't be part of the PR

**Review by nickytonline — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (ryandotfurrer) [reply .env]: Removed in latest commit; thanks for spotting that!

**Review by zeucapua — CHANGES_REQUESTED** [review]
Thank you for the PR! Two things:
1. Please attach screenshots for visual bug fixes like these; the PR template has a section!
2. The mobile view uses a different component called `AddToWorkspaceDrawer` that requires a similar fix: 

Screenshot 2024-09-03 at 07 41 53

**Review by nickytonline — COMMENTED** [review]
Going to take a peek at the fix, but just FYI the .env changes haven't been reverted yet. It still shows as a change in the PR.

CleanShot 2024-09-03 at 10 42 39

**Comment by ryandotfurrer** [comment]
> Thank you for the PR! Two things:
> 
> 1. Please attach screenshots for visual bug fixes like these; the PR template has a section!
> 2. The mobile view uses a different component called `AddToWorkspaceDrawer` that requires a similar fix:

@zeucapua I am so sorry about that! I made a video but it wasn't showing up. I've fixed the PR comment to include the link to the video as well as appropriate screenshots.

The way I went about fixing this touched on the `single-select` component and `AddToWorkspaceModal`, I did not need to touch the `AddToWorkspaceDrawer`. Please let me know your thoughts on the fix and how you would have went about it.

File: .env (hunk 1 above)
Comment (ryandotfurrer) [reply .env]: Did you mean that line or the entire file? I see `.env` in the `.gitignore` file and am surprised it'd include this in the PR.

**Comment by nickytonline** [comment]
Just rerunning the E2E tests we have a bit of flakiness in CI at the moment. 😅

**Comment by nickytonline** [comment]
> Going to take a peek at the fix, but just FYI the .env changes haven't been reverted yet. It still shows as a change in the PR.
> 
> CleanShot 2024-09-03 at 10 42 39

@ryandotfurrer, if you run `git checkout beta -- .env` and commit those changes to your branch and push them, the `.env` file will be removed from your PR.

File: components/atoms/Select/single-select.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ const SingleSelect = ({
       <DropdownMenuTrigger
         data-inset-label={insetLabel}
         className={clsx(
-          "flex text-sm px-3 py-1.5 !border !border-slate-200 rounded-md bg-white data-[state=open]:border-orange-500 min-w-max",
+          "flex w-full md:w-fit text-sm px-3 py-1.5 !border !border-slate-200 rounded-md bg-white data-[state=open]:border-orange-500 min-w-max",
```
Comment (nickytonline) [inline components/atoms/Select/single-select.tsx:63]: I took this for a spin on larger screens and I see the fix, but on smaller screens, there's still a gap between the arrow and the dropdown component. Adding a `justigy-between` should fix it, but take it for a spin on the different screen sizes.

CleanShot 2024-09-03 at 14 30 44

```suggestion
          "flex w-full justify-between md:w-fit text-sm px-3 py-1.5 !border !border-slate-200 rounded-md bg-white data-[state=open]:border-orange-500 min-w-max",
```

CleanShot 2024-09-03 at 14 37 10

**Review by nickytonline — CHANGES_REQUESTED** [review]
Looks like I'm still seeing some issue on smaller screens. I dropped a suggestion for the fix.

**Comment by ryandotfurrer** [comment]
> Just rerunning the E2E tests we have a bit of flakiness in CI at the moment. 😅

We've experienced this in GIS as well, no worries 😅

**Comment by ryandotfurrer** [comment]
> > Going to take a peek at the fix, but just FYI the .env changes haven't been reverted yet. It still shows as a change in the PR.
> > CleanShot 2024-09-03 at 10 42 39
> 
> @ryandotfurrer, if you run `git checkout beta -- .env` and commit those changes to your branch and push them, the `.env` file will be removed from your PR.

Done, ty for that! Any idea why the `.env` file was added in the first place? Just because I made changes to it?  I made changes because Iw as trying to solve the `/login` issue before remembering it's an issue due to GIS.

**Comment by nickytonline** [comment]
> > > Going to take a peek at the fix, but just FYI the .env changes haven't been reverted yet. It still shows as a change in the PR.
> > > CleanShot 2024-09-03 at 10 42 39
> > 
> > 
> > @ryandotfurrer, if you run `git checkout beta -- .env` and commit those changes to your branch and push them, the `.env` file will be removed from your PR.
> 
> Done, ty for that! Any idea why the `.env` file was added in the first place? Just because I made changes to it? I made changes because Iw as trying to solve the `/login` issue before remembering it's an issue due to GIS.

It not in the gitignore for our project as it's settings for people to be able to contribute to the project, so any changes made will appear if you `git add .`.

Maybe longer term we could have an example file and just have a step where we ask people to copy it. Pretty low on the priority list atm though. 😅

**Review by nickytonline — APPROVED** [review]
Looks great in the drawer and in the modal now. Thanks @ryandotfurrer!

CleanShot 2024-09-03 at 15 51 19

CleanShot 2024-09-03 at 15 52 02

**Comment by ryandotfurrer** [comment]
> Looks great in the drawer and in the modal now. Thanks @ryandotfurrer!

Thanks for the collaboration, @nickytonline! I misunderstood the ticket at first but `justify-between` is an obvious solution 😅

I'll ask for clarification next time if there is any area for confusion on the ticket.

**Review by zeucapua — APPROVED** [review]
Great job! LGTM

**Comment by ryandotfurrer** [comment]
> Great job! LGTM

Thank you!

**Review by brandonroberts — APPROVED** [review]
